docs/eventing/samples/sinkbinding/README.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Sink Binding Example"
+linkTitle: "Sink Binding"
+weight: 10
+type: "docs"
+---
+
+A SinkBinding is responsible for linking together "addressable" Kubernetes
+resources that may receive events (aka the event "sink") with Kubernetes
+resources that embed a PodSpec (as `spec.template.spec`) and want to produce
+events.
+
+The SinkBinding can be used to author new event sources using any of the
+familiar compute abstractions that Kubernetes makes available (e.g. Deployment,
+Job, DaemonSet, StatefulSet), or Knative abstractions (e.g. Service,
```

Comment:
add `CronJob`, since that is used in the sample ?

Reply:
CronJob actually isn't PodSpecable, we are binding the Jobs it creates.